admin/tabs/cluster/memory/MemoryMonitorPanel.ts

```diff
@@ -82,3 +88,41 @@ const bbar = hoistCmp.factory<MemoryMonitorModel>({
         );
     }
 });
+
+const instanceSelect = hoistCmp.factory<MemoryMonitorModel>({
+    render({model}) {
+        const {pastInstances, pastInstance, instanceName} = model,
+            options = [{value: null, label: instanceName, lastUpdated: null}] as any;
+        pastInstances?.forEach(it => {
+            options.push({value: it, label: it.name, lastUpdated: it.lastUpdated});
+        });
+        return fragment(
+            select({
```

Comment:
Should we disable this dropdown if you have no past instances?

Reply:
Also, it seems a little bit weird that this control offers you **selected** current instance + **all** past instances, but to change to a different current instance you need to change the top selection.

I understand how it ended up that way and am **not** suggesting we allow you to switch between running instances here - it just seems a bit strange.

Reply:
totally agree and was vexed by this -- we could revisit at some point -- thought this was somewhat consistent with how we were handling the log panel